src/manifest.rs

```diff
@@ -229,13 +241,21 @@ impl str::FromStr for Manifest {
         let mut parser = toml::Parser::new(&input);
 
         parser.parse()
-              .ok_or(parser.errors.pop())
+              .ok_or(format_parse_error(parser))
```

Comment:
Sorry, I forgot to mention this earlier: [Clippy](https://github.com/Manishearth/rust-clippy/) suggests using `.ok_or_else(|| format_parse_error(parser))` instead (you can read more about the suggestion [here](https://github.com/Manishearth/rust-clippy/wiki#or_fun_call)).